packages/preview/sheen-id/0.1.0/typst.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "sheen-id"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Sheen.id"]
+license = "MIT"
+description = "Typst template for sheen.id."
```

Comment:
The word `Typst` is somewhat redundant here, could you remove it? See https://github.com/typst/packages/blob/main/docs/manifest.md#writing-a-good-description